serverless/architecture/serverless-serving-architecture.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-serving-architecture"]
+= Knative Serving architecture
+:context: serverless-serving-architecture
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Serving supports deploying and managing serverless applications, by providing a set of objects as Kubernetes CustomResourceDefinitions (CRDs) that define and control the behavior of serverless workloads on an {product-title} cluster.
+
+Developers use these CRDs to create custom resource instances that can be used as building blocks to address complex use cases. For example:
+
+* Rapidly deploying serverless containers.
+* Automatically scaling pods.
+// TODO: Add other use cases, more advanced tutorials?
```

Comment:
I don't want to add these to the list until we have some docs that back it up in our OpenShift docs, just wanted to remind myself to do it gradually later :slightly_smiling_face:

Reply:
Cool. There's a new tutorial on Blue-Green-Canary deployments by @kameshsampath https://medium.com/@kamesh_sampath/serverless-blue-green-and-canary-with-knative-kn-ad49e8b6aa54 that can be based as a tutorial for that. That's why I've added that as a proposition.

It would be good to describe as many awesome and complicated use case to drive adoption, even without having docs that covers everything.

Reply:
Cool. There's a new tutorial on Blue-Green-Canary deployments by @kameshsampath https://medium.com/@kamesh_sampath/serverless-blue-green-and-canary-with-knative-kn-ad49e8b6aa54 that can be based as a tutorial for that. That's why I've added that as a proposition.

It would be good to describe as many awesome and complicated use case to drive adoption, even without having docs that covers everything.

Reply:
Can you please open a Jira docs issue to document blue green deployments properly so we can scope it in a sprint, and include that link in the description?
I don't think there's a lack of information out there about this to write it up, but it's not part of the scope of this PR, and I'd rather document tutorials etc properly in our docs than point to a 3rd party link that could go out of date, be removed, etc.